src/PageLayout/PageLayout.tsx

```diff
@@ -229,8 +229,21 @@ const contentWidths = {
 
 const Content: React.FC<PageLayoutContentProps> = ({width = 'full', children, sx = {}}) => {
   return (
-    <Box sx={merge<BetterSystemStyleObject>({order: REGION_ORDER.content, flexGrow: 1}, sx)}>
-      <Box sx={{maxWidth: contentWidths[width], marginX: 'auto'}}>{children}</Box>
+    <Box
+      sx={merge<BetterSystemStyleObject>(
+        {
+          order: REGION_ORDER.content,
+          // Set flex-basis to 0% to allow flex-grow to control the width of the content region.
+          // Without this, the content region could wrap onto a different line
+          // than the pane region on wide viewports if its contents are too wide.
+          flexBasis: '0%',
```

Comment:
Minor nitpick: we could just say `flexBasis: 0`. We don't need the unit

Reply:
good catch!